lib/rust/ensogl/core/src/display/scene.rs

```diff
@@ -405,6 +405,12 @@ impl Mouse {
         let position = Vector2(new_pos.x as f32, new_pos.y as f32) - shape.center();
         self.frp.position.emit(position);
     }
+
+    /// Emulate click on background for testing purposes.
+    pub fn click_on_background(&self) {
+        self.target.set(PointerTarget::Background);
+        self.frp.up.emit(frp::io::mouse::Button::Button0);
+    }
```

Comment:
If this is test-only, move it to test-only module. (in the same file). I would also change the api to `scene::test::click_on_background(&scene)`. Then we can keep all test apis nicely layouted

Reply:
Moved it to a separate module with an extension trait.